packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -4,28 +4,61 @@
         <nui-chart class="w-100" [chart]="chartAssist.chart"></nui-chart>
     </div>
     <div class="d-flex" *ngIf="hasLegend()"
-         [ngClass]="[legendShouldBeAlignedRight() ? 'legend scrollable' : 'legend-bottom']">
+                [ngClass]="[legendShouldBeAlignedRight() ? 'legend scrollable' : 'legend-bottom']">
         <nui-legend class="m-auto"
                     [active]="chartAssist.isLegendActive"
                     [interactive]="true"
                     [orientation]="legendShouldBeAlignedRight() ? 'vertical' : 'horizontal'">
-            <nui-legend-series *ngFor="let legendSeries of chartAssist.legendSeriesSet; trackBy: chartAssist.seriesTrackByFn"
-                               [descriptionPrimary]="legendSeries.legendDescriptionPrimary"
-                               [descriptionSecondary]="legendSeries.legendDescriptionSecondary"
-                               [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
-                               (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)"
-                               (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
-                               (mouseleave)="chartAssist.resetVisibleSeries()"
-                               [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
-                               (click)="onLegendClick(legendSeries, $event)">
+            <nui-legend-series 
+                    [ngClass]="'legend-container '+ 'state-' + chartAssist.renderStatesIndex[legendSeries.id]?.state"
```

Comment:
Do you mind trying to use the `[seriesRenderState]` input now that we've made the styles public? The nui-legend-series should add the correct render state classes based on the chart assist's renderStatesIndex.

Here's a usage of that:
https://github.com/solarwinds/nova/blob/0c3bd4215aa7054e2488b02de9acf19921bd7bde/packages/charts/examples/components/demo/advanced-usage/legend/legend-interactive/legend-interactive.example.component.html#L19